scripts/srt-dev.lua

```diff
@@ -0,0 +1,887 @@
+-- @brief srt-dev Protocol dissector plugin
+
+-- create a new dissector
+local NAME = "SRT-dev"
+local srt_dev = Proto(NAME, "SRT-dev Protocol")
+
+-- create a preference of a Protocol
+srt_dev.prefs["srt_udp_port"] = Pref.uint("SRT UDP Port", 1935, "SRT UDP Port")
+
+-- create fields of srt_dev
+-- Base.HEX, Base.DEC, Base.OCT, Base.UNIT_STRING, Base.NONE
+local fields = srt_dev.fields
+-- General field
+local pack_type_select = {
+	[0] = "Data Packet",
+	[1] = "Control Packet"
+}
+fields.pack_type_tree = ProtoField.uint32(NAME .. ".pack_type_tree", "Packet Type", base.HEX)
+fields.pack_type = ProtoField.uint16("srt_dev.pack_type", "Packet Type", base.HEX, pack_type_select, 0x8000)
+fields.reserve = ProtoField.uint16("srt_dev.reserve", "Reserve", base.DEC)
+fields.additional_info = ProtoField.uint32("srt_dev.additional_info", "Additional Information", base.DEC)
+fields.time_stamp = ProtoField.uint32("srt_dev.time_stamp", "Time Stamp", base.DEC)
+fields.dst_sock = ProtoField.uint32("srt_dev.dst_sock", "Destination Socket ID", base.DEC)
+fields.none = ProtoField.none("srt_dev.none", "none", base.NONE)
+
+-- Data packet fileds
+fields.data_flag_info_tree = ProtoField.uint8("srt_dev.data_flag_info_tree", "Data Flag Info", base.HEX)
+local FF_state_select = {
+	[0] = "[Middle packet]",
+	[1] = "[Last packet]",
+	[2] = "[First packet]",
+	[3] = "[Single packet]"
+}
+fields.FF_state = ProtoField.uint8("srt_dev.FF_state", "FF state", base.HEX, FF_state_select, 0xC0)
+local O_state_select = {
+	[0] = "[Data delivered unordered]",
+	[1] = "[Data delivered in order]"
```

Comment:
The meaning of this field is that when clear then a message that is complete and is waiting in the buffer, but is blocked by an incomplete message (due to lost packets waiting for recovery), can be delivered before the blocking message is completed.

Short names I selected somewhere in the code for that is: ORD_REQUIRED and ORD_RELAXED.